src/dict.c

```diff
@@ -510,20 +510,23 @@ void dictRelease(dict *d)
 dictEntry *dictFind(dict *d, const void *key)
 {
     dictEntry *he;
-    uint64_t h, idx, table;
+    uint64_t h;
+    long idx_0, idx_1;
 
     if (dictSize(d) == 0) return NULL; /* dict is empty */
     if (dictIsRehashing(d)) _dictRehashStep(d);
     h = dictHashKey(d, key);
-    for (table = 0; table <= 1; table++) {
-        idx = h & DICTHT_SIZE_MASK(d->ht_size_exp[table]);
-        he = d->ht_table[table][idx];
-        while(he) {
-            if (key==he->key || dictCompareKeys(d, key, he->key))
-                return he;
-            he = he->next;
-        }
-        if (!dictIsRehashing(d)) return NULL;
+    idx_0 = h & DICTHT_SIZE_MASK(d->ht_size_exp[0]);
+    if (likely(d->rehashidx < idx_0)){
```

Comment:
```suggestion
    if (likely(d->rehashidx < idx_0)) {
```

Reply:
i have a new pr in 9898[https://github.com/redis/redis/pull/9898](url), because i delete origin repositories and can't recover it. does it have chance to merge?